checklists/alz_checklist.en.json

```diff
@@ -1491,7 +1541,7 @@
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "Firewall",
-            "text": "Integrate Azure Firewall with Azure Monitor and enable diagnostic logging to store and analyze firewall logs.",
+            "text": "Integrate Azure Firewall with Azure Monitor and enable diagnostic logging to store and analyze firewall logs and metrics.",
             "waf": "Operations",
```

Comment:
Hi Igor, it seems to miss an id tag in this JSON entry. not sure if this is relevant for this change, but you can make the call to have it added now or later.

Reply:
Hi @hgotar1, ID is there on line 1548